src/app/portlets/dot-porlet-detail/dot-workflow-task/dot-workflow-task.component.spec.ts

```diff
@@ -99,4 +99,14 @@ describe('DotWorkflowTaskComponent', () => {
         });
         expect(component.onCloseWorkflowTaskEditor).toHaveBeenCalledTimes(1);
     });
+
+    it('should redirect to /workflow when close event is triggered', () => {
+        spyOn(component, 'onCloseWorkflowTaskEditor');
```

Comment:
If I remove the body of the method `onCloseWorkflowTaskEditor` this test will keep passing but the UI will fail.